docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+services:
```

Comment:
lets not have docker compose at the root of the repo. I think this was moved to tooling/sync